src/views/monitoring/alert/modules/alert-list/AlertDataTable.vue

```diff
@@ -13,10 +13,12 @@
             :select-index.sync="selectIndex"
             :sort-by.sync="sortBy"
             :sort-desc.sync="sortDesc"
+            :this-page.sync="thisPage"
+            :page-size.sync="pageLimit"
```

Comment:
강제로 특정 값으로 주입하고자 함이 아니면, props 를 바인딩하지 않아도 좋습니다!
```suggestion
```
다만, api 요청 쿼리의 page limit 이 기본 page size 과 같아야 하는데, toolbox table 의 기본 page size 가 무엇인지 몰라 헷갈릴 수는 있을 것 같아요..
그러면 아래처럼 해도 좋을 것 같아요.
```suggestion
            :page-size="15"
```
참고로 toolbox table 의 기본 page size 는 15 이기 때문에, 첫 번째 제안드린 것과 동작은 같습니다.

```suggestion
            :page-size="pageLimit"
```
아니면 pageLimit 변수를 그대로 두고, 굳이 sync 를 하지 않아도 될 것 같아요.
3 가지 중 가독성이 높다고 판단되는 것으로 해주셔도 될 것 같아요.

지금도 동작에는 이상이 없지만, 불필요한 양방향 바인딩이 있는 것 같아서 말씀드렸어요~ (이렇게 되어있는 레거시 페이지들이 많아서 혼란스러우실 것 같아 설명을 좀 길게 드립니당)

---
toolbox table 의 sync를 지원하는 모든 props는
1. props 가 있으면, 그 값으로 초기화합니다.
2. sync 로 양방향 바인딩 되어있으면, props 값을 바탕으로 동작합니다.
3. sync 로 양방향 바인딩 되어있지 않으면, 위에서 받은 props 값을 신뢰하기 어려우므로, 내부 변수로 동작합니다.
4. 만약  sync 로 앙방향 바인딩 안된 경우(단방향)에, props 로 주입된 값이 변경되면, 강제로 해당 props 값을 변경해야 하는 것으로 간주하여, 내부 변수를 변경된 props 값으로 바꿉니다.